apm-agent-plugins/apm-logging-plugin/apm-logging-plugin-common/src/test/java/co/elastic/apm/agent/loginstr/LoggingInstrumentationTest.java

```diff
@@ -82,71 +95,70 @@ public void setup() throws Exception {
 
         // IMPORTANT: keep this last, so that it doesn't interfere with Mockito settings above
         serviceName = Objects.requireNonNull(tracer.getMetaDataFuture().get(2000, TimeUnit.MILLISECONDS).getService().getName());
+
+        transaction = Objects.requireNonNull(tracer.startRootTransaction(null)).activate();
```

Comment:
[minor] could be replaced with a call to `startRootTransaction()`

Reply:
I changed to using `startTestRootTransaction()` instead, assuming that this is what you meant. Let me know it was something else.